### PR TITLE
Add covermode

### PR DESCRIPTION
**Pull request by seangerhardt-wf**
## **Problem**: 
Can't specify the covermode. 

The cover mode needs to be set to `atomic` if you want to run with `race`.

## **Solution**:
Allow ability to pass in cover mode, with default value of `count`, which is what's currently assumed.

@wesleybalvanz-wf @markcampanelli-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 0</summary>

</details>

File: goverge/test/test_main.py [hunk 1]
```diff
@@ -117,6 +118,11 @@ def test_custom(self):
         args = main._parse_args(['--go_flags=-x', '--go_flags=-timeout=5m'])
         self.assertEqual(["-x", "-timeout=5m"], vars(args).get('go_flags'))
 
+    def test_covermode(self):
+        args = main._parse_args(['--covermode=atomic'])
+        print vars(args)
```
Comment (wesleybalvanz-wf) [inline goverge/test/test_main.py]: Might want to remove the print here. :)

**Comment by wesleybalvanz-wf** [comment]
Looks like you have some flake8 errors that you are hitting in the ci.

**Comment by seangerhardt-wf** [comment]
@wesleybalvanz-wf Issues should be addressed

**Comment by aldenpeterson-wf** [comment]
lgtm

**Comment by wesleybalvanz-wf** [comment]
+1

**Pull request merged**